src/PhpOfBy/WebsiteBundle/Resources/config/services.yml

```diff
@@ -6,4 +6,3 @@ services:
     class: '%phpofby_website.controller.default.class%'
     parent: common.contoller.abstract
     arguments: []
-    lazy: true
```

Comment:
Because after installing doctrine migrations bundle we have problems with `@MVC\Template` annotation. Symfony cannot properly read and apply this annotations.